mlflow/pyfunc/model.py

```diff
@@ -382,17 +386,27 @@ def _save_model_with_class_artifacts_params(
             shutil.move(tmp_artifacts_dir.path(), os.path.join(path, saved_artifacts_dir_subpath))
         custom_model_config_kwargs[CONFIG_KEY_ARTIFACTS] = saved_artifacts_config
 
-    if streamable is None:
+    if python_model and streamable is None:
         streamable = python_model.__class__.predict_stream != PythonModel.predict_stream
```

Comment:
Doesn't this mean that we won't support code-based models as streamable? Streaming is super important for model deployment of GenAI apps, particularly on Databricks.

Can we support this and add a test case? Here's an idea: pass `python_model` *and* `model_code_path` to `_save_model_with_class_artifacts_params`. Update `_save_model_with_class_artifacts_params` to only save `model_code_path` if specified (don't save the `python_model` in that case).